tests/testthat/test-label-miss-1d.R

```diff
@@ -15,16 +15,16 @@ test_that("label_miss_1d returns a vector of the same length as the input",{
 
 test_that("label_miss_1d returns character vector",{
```

Comment:
(suggestion) perhaps modify the description to reflect that this is a factor, and maybe a comment that `typeof(factor())` is "integer".

Reply:
Great catch, I overlooked that description. Its fixed